web/src/components/SubmissionsTable.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-data-table
+    class="submissions-table row-pointer"
+    :headers="headers"
+    :items="items"
+    :search="searchValue"
+    sort-by="similarity"
+    sort-desc
+    hide-default-footer
+    disable-pagination
+    must-sort
+    fixed-header
+    @click:row="rowClicked"
+  >
+    <template #item.name="{ item }">
+      <div class="submission-info">
+        <div class="submission-name">
+          {{ item.name }}
+        </div>
+
+        <div class="submission-path text--secondary">
+          <v-icon small>mdi-file-document-outline</v-icon>
+          <span>{{ item.path }}</span>
+        </div>
+      </div>
+    </template>
+
+    <template #item.label="{ item }">

```

Comment:
Can the label row be hidden if there are no labels present in the dataset? I think it could be confusing to show "No Label", a user could expect some submissions to have a label.